cmd/client/cli/commands.go

```diff
@@ -396,20 +401,25 @@ func kubeConfig() *cobra.Command {
 			kubefile := fmt.Sprintf("%s/.kube/config", home)
 			log.Printf("reading existing kube config : %s\n", kubefile)
 
+			skipMerge := false
 			currentKC, err := clientcmd.LoadFromFile(kubefile)
-			if err != nil {
-				log.Fatalf("failed to load the existing kube config : %s\n", err.Error())
+
+			if os.IsNotExist(err) {
+				//file does not exist, create new one
+				skipMerge = true
+
 			}
 
-			newConfig, err := clientcmd.Load(res.GetConfig())
-			if err != nil {
-				log.Fatalf("failed to read kube config : %s\n", err.Error())
+			if !skipMerge && err != nil {
+				log.Fatalf("failed to load the existing kube config : %s\n", err.Error())
 			}
 
 			kubeConfg := clientcmdapi.NewConfig()
 			//set the current cluster context as new context
 			newConfig.CurrentContext = res.ClusterName
-			mergo.Merge(kubeConfg, currentKC, mergo.WithOverride)
+			if !skipMerge {
+				mergo.Merge(kubeConfg, currentKC, mergo.WithOverride)
+			}
 			mergo.Merge(kubeConfg, newConfig, mergo.WithOverride)
```

Comment:
Should we not remove this?

Reply:
looked cleaner in a way taht we are merging existing and new cluster config to new kube config created in line 417

Reply:
cleaned this up.